packages/assets-controllers/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.0.2]
+### Changed
+- Relax dependencies on `@metamask/base-controller`, `@metamask/controller-utils`, `@metamask/network-controller`, and `@metamask/preferences-controller` (use `^` instead of `~`) ([#998](https://github.com/MetaMask/controllers/pull/998))
+- Align with changes to `@metamask/network-controller`'s `NetworkState` type (`provider` -> `providerConfig`) ([#995](https://github.com/MetaMask/controllers/pull/995))
```

Comment:
This part is a breaking change because we're changing the expectation/type for the `onNetworkStateChange` constructor parameter. This would break on the extension for example, unless we updated the callback we pass in to accommodate this.

Reply:
Looks like the breaking changes are:

* Constructor parameter `onNetworkStateChange` for the following controllers now expects network state to have `providerConfig` rather than `provider` :
  * `AssetsContractController`
  * `NftController`
  * `NftDetectionController`
  * `TokenDetectionController`
  * `TokenListController`
  * `TokenRatesController`
  * `TokenController`
* `TokenDetectionController` constructor parameter `getNetworkState` now expected to return state with `providerConfig` rather than `provider`

Reply:
Yeah... you're right. Thanks for checking this. I'll make the changes tomorrow.

Reply:
Done in [a24e1cc](https://github.com/MetaMask/controllers/pull/999/commits/a24e1ccf93fe97dd79dcba59200b3a43bdd4bf95).